nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/image/CropAndResize.java

```diff
@@ -89,8 +91,8 @@ public void initFromTensorFlow(NodeDef nodeDef, SameDiff initWith, Map<String, A
     }
 
     protected void addArgs() {
-        iArguments.clear();
-        tArguments.clear();
+        //iArguments.clear();
+        //tArguments.clear();
```

Comment:
Checking code, it should be safe to remove this. Let's actually delete it then :)

Reply:
Fixed.